aws-typescript/tests/infra.spec.ts

```diff
@@ -41,11 +41,20 @@ describe("infrastructure", () => {
     });
 
     // Example test. To run, uncomment and run `npm test`.
-    // describe("bucket", () => {
-    //     it("must have a name tag", async () => {
-    //         const tags = await promiseOf(infra.bucket.tags);
-    //         expect(tags).toBeDefined();
-    //         expect(tags).toHaveProperty("Name");
-    //     });
-    // });
+    describe("bucket", () => {
+        it("must have a name tag", async () => {
+            const tags = await promiseOf(infra.bucket.tags);
+            expect(tags).toBeDefined();
+            expect(tags).toHaveProperty("Name");
+        });
+        it("must have the right tags", async () => {
+            await (expect(infra.bucket.tags) as any).toEqualOutputOf({ "Name": "My bucket oops" })
```

Comment:
I didn't extend the `expect` type to know about our matchers, with that we won't need the cast to `any` here.